src/data/Profile.json

```diff
@@ -836,5 +836,18 @@
       "GitHub": "https://github.com/CyberManMatt>",
       "LinkedIn": "https://www.linkedin.com/in/matthew-jarrett-33605663/"
     }
- }
+ },
+ {
+  "name": "Sudipta Pradhan",
+  "location": "Bhubaneswar, Odisha, India",
+  "bio": "Final Year B.Tech Undergrad just on the verge of completing the degree 💁‍♀️ Front-End web developer since 3 years ✨",
+  "avatar": "https://avatars.githubusercontent.com/u/64891121?v=4",
+  "portfolio": "",
+  "skills": ["HTML", "CSS", "Javascript", "Bootstrap", "ReactJS", "NextJS"],
+  "social": {
+    "GitHub": "https://github.com/sudiptapradhan",
+    "Twitter": "https://twitter.com/_SudiptaPradhan",
+    "LinkedIn": "https://www.linkedin.com/in/sudipta-pradhan-33564a1b4/"
+  }
+},
```

Comment:
```suggestion
}
```

Reply:
Hey @sudiptapradhan if you adding your profile at last than `,` comma is not required, remove the comma and commit the changes.